docs/docs/realtime.md

```diff
@@ -0,0 +1,689 @@
+
+#### Using Stream
+
+The '@stream' directive allows you to stream the individual items of a field of the list type as the items are available.
+
+:::info
+The `@stream` directive is currently **not** supported by Apollo GraphQL client.
+:::
+
+## Features
+
+RedwoodJS Realtime handles the hard parts of a GraphQL Realtime implementation by automatically:
+
+- allowing GraphQL Subscription operations to be handled
+- merging in your subscriptions types and mapping their handler functions (subscribe, and resolve) to your GraphQL schema letting you keep your subscription logic organized and apart from services (your subscription my use a service to respond to an event)
+- authenticating subscription requests using the same `@requireAuth` directives already protecting other queries and mutations (or you can implement your own validator directive)
+- adding in the `@live` query directive to your GraphQL schema and setting up the `useLiveQuery` envelop plugin to handle requests, invalidation, and managing the storage mechanism needed
+- creating and configuring in-memory and persisted Redis stores uses by the PubSub transport for subscriptions and Live Queries (and letting you switch between them in development and production)
+- placing the pubSub transport and stores into the GraphQL context so you can use them in services, subscription resolvers, or elsewhere (like a webhook, function, or job) to publish an event or invalidate data
+- typing you subscription channel event payloads
+- support `@defer` and `@stream` directives
+
+It provides a first-class developer experience for real-time updates with GraphQL so you can easily
+
+- respond to an event (e.g. NewPost, NewUserNotification)
+- respond to a data change (e.g. Post 123's title updated)
+
+and have the latest data reflected in your app.
+
+Lastly, the Redwood CLI has commands to
+
+- generate a boilerplate implementation and sample code needed to create your custom
+    - subscriptions
+    - live Queries
+
+Regardless of the implementation chosen, **a stateful server and store are needed** to track changes, invalidation, or who wants to be informed about the change.
+
+### What can I build with Realtime?
+
+- Application Alerts and Messages
+- User Notifications
+- Live Charts
+- Location updates
+- Auction bid updates
+- Messaging
+- OpenAI streaming responses
+
+## RedwoodJS Realtime Setup
+
+To setup Realtime in an existing RedwoodJS project, run the following commands:
+
+* `yarn rw exp setup-server-file`
+* `yarn rw exp setup-realtime`
+
+You will get:
+
+* `api/server.ts` where you configure your Fastify server and GraphQL
+* `api/lib/realtime.ts` where you consume your subscriptions and configure realtime with an in-memory or Redis store
+* The auction, countdown, and chat examples. You'll find sdl, services and subscriptions for each. There is no UI setup for these examples. You can find information on how to try them out using the GraphiQL playground.
```

Comment:
And now you also get stream and defer examples?

Reply:
Yes, one does stream and defer examples. I'll update that, thanks!